test/test-cid.js

```diff
@@ -392,6 +392,13 @@ describe('CID', () => {
     assert.ok(equals(json.hash, hash.bytes))
   })
 
+  it('truncate()', async () => {
```

Comment:
Sidenote: we should hace additional  test that document how this works for  "the shortest CID possible" which is https://cid.ipfs.io/#bafkqaaa